spec/controllers/users/invitations_controller_spec.rb

```diff
@@ -38,9 +40,10 @@
       end
 
       it 'does not allow you to invite admins as a manager' do
-        expect {
-          post :create, user: admin_invitation
-        }.to raise_error 'Unprivileged invitation error: Non-admin user is inviting an admin.'
+        post :create, user: admin_invitation
+        expect(response).to redirect_to(new_user_invitation_path)
+        get :new
+        expect(response.body).to include 'You cannot create an admin account'
```

Comment:
You are right. I figured that if the error message content is present, then... Thank you